hm.tex

```diff
@@ -509,7 +511,7 @@ \section{Hamilton's variational principle}\label{sec:varpri}
 \end{corollary}
 
 \begin{proof}
-    For a non--degenerate lagrangian, the Euler-Lagrange equations can be rewritten in the form
+    For a non--degenerate lagrangian, the Euler-Lagrange equations can be rewritten in the form %by the implicit function theorem?
```

Comment:
By computing the total derivative in the E-L equations and inverting the matrix `\Lambda` which appears as prefactor of the second derivatives
```suggestion
    For a non--degenerate lagrangian, a direct computation shows that the Euler-Lagrange equations can be rewritten in the form
```

Reply:
I think my issue is the Einstein notation specifically. We did not encounter it in AoM with Gert Vegter, nor am I young enough to have taken Multivariate analysis with Roland van der Veen. So I have a hard time following the indices. I think another students should look at this, or keep this question for the next batch of students next year

Reply:
Oh yes, that is probably easier for students that took the new multivariable calculus or analysis on manifolds. It is worth adding a remark on Einstein's notation here though. I went into details on it in the recorded lecture, which may also help.